### ISSUE TITLE
fk_pipe in valves gets filled with bogus pipe id's

### ISSUE THREAD

**Issue by tudorbarascu** [qwat/qwat-data-model#76]
the title is explicit enough.

**Comment by tudorbarascu** [comment]
It's not fixed yet, you can test by trying 

```
INSERT INTO qwat_od.distributor (name) VALUES ('Demo Distributor');

INSERT INTO qwat_od.pipe (geometry, fk_function, fk_material, fk_distributor, fk_status, fk_watertype, fk_installmethod, fk_precision, fk_bedding)
SELECT ST_SetSRID(ST_GeomFromText('LINESTRING (559980 149980, 559990 149990, 560000 150000)'), 21781), 4105, 101, 1, 1301, 1502, 101, 101, 101;

INSERT INTO qwat_od.pipe (geometry, fk_function, fk_material, fk_distributor, fk_status, fk_watertype, fk_installmethod, fk_precision, fk_bedding)
SELECT ST_SetSRID(ST_GeomFromText('LINESTRING (560000 150000, 560010 150010, 560020 150020)'), 21781), 4105, 101, 1, 1301, 1502, 101, 101, 101;

INSERT INTO qwat_od.pipe (geometry, fk_function, fk_material, fk_distributor, fk_status, fk_watertype, fk_installmethod, fk_precision, fk_bedding)
SELECT ST_SetSRID(ST_GeomFromText('LINESTRING (560000 150000, 559990 150010, 559980 150020)'), 21781), 4105, 101, 1, 1301, 1502, 101, 101, 101;

INSERT INTO qwat_od.vw_element_valve (geometry, fk_distributor, fk_status, fk_valve_type, fk_valve_function, fk_actuation)
SELECT ST_SetSRID(ST_GeomFromText('POINT(559990 149990)'), 21781), 1, 1301, 101, 101, 101;

INSERT INTO qwat_od.vw_element_valve (geometry, fk_distributor, fk_status, fk_valve_type, fk_valve_function, fk_actuation)
SELECT ST_SetSRID(ST_GeomFromText('POINT(559990 150010)'), 21781), 1, 1301, 101, 101, 101;
```
Which creates the following setup:
![pic](https://cloud.githubusercontent.com/assets/3179646/10841355/47942eb2-7ef1-11e5-9cca-7a2b9c859387.png)

The pipes start end points are:
- 1-2
- 2-3
- 2-4

The pipe id for both valves is 1, which is incorrect.
Also, the rotations of the pipes are currently:
-  `-45` for the one on pipe 1
- `-135` for the one on pipe 3

...

**Comment by tudorbarascu** [comment]
Thanks a lot for investigating Denis ;).

**Comment by 3nids** [comment]
should be good now!

**Comment by 3nids** [comment]
so, orientation is ok for you now?

**Comment by tudorbarascu** [comment]
It looks good now regarding fk_pipe
Orientation is bad though.
```
SELECT id, _pipe_orientation FROM qwat_od.vw_element_valve;
```
outputs
```
 id | _pipe_orientation 
----+-------------------
  5 |               -45
  6 |              -135
```
When in fact it should output:
```
 id | _pipe_orientation 
----+-------------------
  5 |               45
  6 |              315
```

**Comment by 3nids** [comment]
There is a 90° shift.
It is somehow practical because there is no need for an expression in QGIS (it uses directly the value of the angle) for the rotation.
Is this ok for you?

**Comment by 3nids** [comment]
I might have fixed some errors though 719873b

**Comment by tudorbarascu** [comment]
Good morning.

It's ok by me if it's shifted with 90 degrees. 
However, I think it would be better if the SVG's/symbols we're using are rotated using Inkscape.
This way:
- the orientation is respected
- nobody will ask us why it's shifted
- we won't have to put any expression
- easier to debug
- we avoid making non-necessary conventions (90 degrees shift in rotation)

I still have some strange rotations, will try and make a test case.
Thanks a lot!
